server/src/main/resources/banner.txt

```diff
@@ -0,0 +1,10 @@
+ _                 _______ _________ _______  _______  _______ 
+| \    /\|\     /|(  ____ )\__   __/(  ___  )(       )(  ___  )
+|  \  / /| )   ( || (    )|   ) (   | (   ) || () () || (   ) |
+|  (_/ / | |   | || (____)|   | |   | (___) || || || || (___) |
+|   _ (  | |   | ||     __)   | |   |  ___  || |(_)| ||  ___  |
+|  ( \ \ | |   | || (\ (      | |   | (   ) || |   | || (   ) |
+|  /  \ \| (___) || ) \ \__   | |   | )   ( || )   ( || )   ( |
+|_/    \/(_______)|/   \__/   )_(   |/     \||/     \||/     \|
+${application.title} (v${application.version})
```

Comment:
```suggestion
${application.title} ${application.formatted-version}
```

Spring itself provides a variable for `banner.txt` to get a formatted version. See the [docs](https://docs.spring.io/spring-boot/reference/features/spring-application.html#features.spring-application.banner) for more information. Beware, this only works with the jar and not when started from within the IDE.